py/visdom/server.py

```diff
@@ -1655,6 +1663,9 @@ def download_scripts(proxies=None, install_dir=None):
         # Stanford Javascript Crypto Library for Password Hashing
         '%ssjcl@1.0.7/sjcl.js' % b: 'sjcl.js',
         '%slayout-bin-packer@1.4.0/dist/layout-bin-packer.js.map' % b: 'layout-bin-packer.js.map',
+        'http://d3js.org/d3.v3.min.js' : 'd3.v3.min.js',
```

Comment:
Nit - it would be reasonable to include a comment above these scripts about what they're imported for.

Reply:
Done!